src/components/page/JoinNow.svelte

```diff
@@ -81,7 +88,22 @@
     flex-direction: row;
     align-items: center;
     color: #00a900;
-    margin-bottom: 30px;
+    margin-bottom: 1.75rem;
+    font-size: 1.25rem;
+  }
+
+  .info-block{
+    margin-bottom: 1.75rem;
+    &__heading{
```

Comment:
@zbrody I see you do this css trick a lot instead of something like below. 

```
<div class="info-block">
    <p class="heading">What you'll need</p>
    <ul class="list">
      <li>Your Tax File Number (TFN)</li>
      <li>Your mobile phone</li>
    </ul>
  </div>

.info-block {
    .heading {}
    .list {}
```

Is there a particular reason for this? i.e Is this to make sure there is no chance of a heading or list style already defined that maybe affect the design?
I can see that your technique can prevent that, however svelte scopes css to the component so the chances of this happening are less likely.